docs/user/how-to.md

```diff
@@ -46,3 +46,44 @@ You can now say eg `"echo air past bat"`.
 See the [Talon-side api docs](./customization.md#cursorless-public-api) for more on creating custom Cursorless commands
 
 :::
+
+## How do I run a custom Python transformation on a target?
+
+1. Add the transformation to a Python file in your Talon user directory:
+
+   ```python
+    from talon import Module
+
+    mod = Module()
+
+
+    @mod.action_class
+    class Actions:
+        def hello(text: str) -> str:
+            """Returns a greeting for the given text."""
+            return f"Hello, {text}!"
+   ```
+
+2. Add a spoken form to your `vscode.talon`:
+
+   ```talon
+   hello <user.cursorless_target>:
+       old = user.cursorless_get_text(cursorless_target, 1)
```

Comment:
Doesn't `True` work on public Talon?

Reply:
didn't work for me 🤔

Reply:
It's already used in community actually. Note that in Talon script its true with lower case t. ie not same case as python.
https://github.com/talonhub/community/blob/93c9261c240078c46290ae1a500f2c9cfc749833/settings.talon#L34

Reply:
Yeah I tried with lowercase. Maybe I was doing something dumb. I'll try again